src/mqtt.cpp

```diff
@@ -107,6 +108,43 @@ void mqttmsg_callback(MongooseString topic, MongooseString payload) {
       divertmode_update(newdivert);
     }
   }
+  else if (topic_string == mqtt_topic + "/charge_current/set")
```

Comment:
This does not match the subscribed topic:

```suggestion
  else if (topic_string == mqtt_topic + "/pilot/set")
```

Reply:
Also the GitHub workflows look to have been broken by Platform IO so there is a dependency on #207

Reply:
you mean I can merge #207 in my base main ?